### ISSUE TITLE
"learning rate" vs "lr"

### ISSUE THREAD

**Issue by OverLordGoldDragon** [keras-team/keras#13393]
<s>`tf.keras` still uses `lr`</s>, and F. Chollet noted Keras' development onward will orient around `tf.keras` - this said, Why was `lr` renamed to `learning_rate` in Keras 2.3.0? I doubt much clarity is lost as `lr` is a universal parameter in machine learning, and I personally prefer `lr`. More importantly, I'm making a custom optimizer, and <s>this difference complicates compatibility with `keras` and `tf.keras`.</s>

<hr>

**UPDATE**: `tf.keras` does _not_ use `lr` in TF2; I may have confused it with `tf.python.keras`. Moreover, `learning_rate` is backwards compatible as it's set via `kwargs.get('lr', learning_rate)`.

**Comment by kseniakhakalo** [comment]
Now ReduceLROnPlateau callback doesn't work, since it expects "lr" attribute from the optimizer. 
<img width="667" alt="image" src="https://user-images.githubusercontent.com/15153264/67615652-c6fcdb00-f7d7-11e9-943a-1b6e1e143e9c.png">

**Comment by OverLordGoldDragon** [comment]
Updated; still interested in rationale to switching to `learning_rate`.

**Comment by wuzhixin1010** [comment]
![image](https://user-images.githubusercontent.com/55953862/68260807-0c8f9280-0079-11ea-8911-2edeac35a8b5.png)
Hello.
I change my keras version to 2.2.5,but the ReduceLROnPlateau callback still doesn't work.
How should I solve this problem?
